### PR TITLE
Use configurable callback_url

### PR DESCRIPTION
**Pull request by elfosardo**
Adding bash script as entrypoint to be able to configure the
callback_url using custom ip and port.
It defaults to mdns.

### CODE REVIEW

**Comment by dtantsur** [comment]
Did you squash the two commits here? I'd suggest you keep them separate (even if in the same PR).

**Comment by elfosardo** [comment]
> Did you squash the two commits here? I'd suggest you keep them separate (even if in the same PR).

@dtantsur this is actually just one commit.
If you mean separate the changes to Dockerfile and Dockerfile.ocp, the Dockerfile change is not needed at all since it's not used in the Openshift image, I can just remove it; the only reason I added it is to keep it as close as possible to metal3-io.

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
```
Comment (dtantsur) [inline Dockerfile]: This change is not from the commit

Comment (elfosardo) [reply Dockerfile]: this is to align the Dockerfile from metal3-io, although it's not necessary as it's not needed for this repo, Dockerfile.ocp is used instead
I can remove the change or just add a note in the commit

**Pull request merged**